ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -808,10 +808,12 @@ KEXT_STATIC int HandleFileOpOperation(
         
         // TODO(#1367): We need to handle failures to lookup the vnode.  If we fail to lookup the vnode
         // it's possible that we'll miss notifications
-        errno_t toErr = vnode_lookup(newPath, 0 /* flags */, &currentVnode, context);
+        errno_t toErr = vnode_lookup(newPath, /* flags: */ VNODE_LOOKUP_NOFOLLOW, &currentVnode, context);
         if (0 != toErr)
         {
-            KextLog_Error("HandleFileOpOperation (KAUTH_FILEOP_RENAME): vnode_lookup failed, errno %d for path '%s'", toErr, newPath);
+            VirtualizationRootHandle providerHandle = ActiveProvider_FindForPath(newPath);
+            // TODO(#1480): Drop log level to KEXTLOG_DEFAULT when providerHandle is 'none' once these failures become rare
+            KextLog_Error("HandleFileOpOperation (KAUTH_FILEOP_RENAME): vnode_lookup failed, errno %d for path '%s'; path lies within root %d%s", toErr, newPath, providerHandle, providerHandle == RootHandle_None ? "" : " with active provider");
```

Comment:
I've added TODO comments here and in the KAUTH_FILEOP_LINK case below while leaving the log message at error level for all cases for now.